### PR TITLE
Ensure that there are no paralellization hickups with xargs

### PR DESCRIPTION
**Pull request by Inventitech**
Refs #10

### CODE REVIEW

**Comment by Inventitech** [comment]
Hi @dreamyguy,

I see there is some good stuff coming, but maybe you can accept this PR in the meantime to remove the parallelization hickups that introducing xargs caused.

File: scripts/output-intermediate-gitlog.sh [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
 
 cd ..
 
 test "$1" || exit 1
 dir=$1
 
-cd $dir &&
-        echo -e "${Whi}Outputting ${Pur}${PWD##*/}${RCol}" >&2 &&
-        git log --all --no-merges --shortstat --reverse --pretty=format:'commits\trepository\t'"${PWD##*/}"'\tcommit_hash\t%H\tcommit_hash_abbreviated\t%h\ttree_hash\t%T\ttree_hash_abbreviated\t%t\tparent_hashes\t%P\tparent_hashes_abbreviated\t%p\tauthor_name\t%an\tauthor_name_mailmap\t%aN\tauthor_email\t%ae\tauthor_email_mailmap\t%aE\tauthor_date\t%ad\tauthor_date_RFC2822\t%aD\tauthor_date_relative\t%ar\tauthor_date_unix_timestamp\t%at\tauthor_date_iso_8601\t%ai\tauthor_date_iso_8601_strict\t%aI\tcommitter_name\t%cn\tcommitter_name_mailmap\t%cN\tcommitter_email\t%ce\tcommitter_email_mailmap\t%cE\tcommitter_date\t%cd\tcommitter_date_RFC2822\t%cD\tcommitter_date_relative\t%cr\tcommitter_date_unix_timestamp\t%ct\tcommitter_date_iso_8601\t%ci\tcommitter_date_iso_8601_strict\t%cI\tref_names\t%d\tref_names_no_wrapping\t%D\tencoding\t%e\tsubject\t%s\tsubject_sanitized\t%f\tcommit_notes\t%N\tstats\t' |
+tempOutputFile="/home/mbeller/gitloggg/inventitech-gitlogg/_tmp/gitlogg.tmp"
```
Comment (dreamyguy) [inline scripts/output-intermediate-gitlog.sh]: I deleted this line while reviewing, but still got the error I posted in the PR.

**Comment by dreamyguy** [comment]
Got the following error:

```
Wallaces-iMac:gitlogg Wallace$ bash ./scripts/gitlogg-generate-log.sh
Info: Calculating in 7 thread(s)
Generating git log for all 4 repositories located at './_repos/*/'. This might take a while!
Outputting git.git
Outputting core.git
Outputting linux.git
Outputting odoo.git
./scripts/output-intermediate-gitlog.sh: line 30: _tmp/gitlogg.tmp.part.: No such file or directory
./scripts/output-intermediate-gitlog.sh: line 30: _tmp/gitlogg.tmp.part.: No such file or directory
./scripts/output-intermediate-gitlog.sh: line 30: _tmp/gitlogg.tmp.part.: No such file or directory
./scripts/output-intermediate-gitlog.sh: line 30: _tmp/gitlogg.tmp.part.: No such file or directory
cat: _tmp/gitlogg.tmp.part.*: No such file or directory
rm: _tmp/gitlogg.tmp.part.*: No such file or directory
The file _tmp/gitlogg.tmp generated in: 0s
Parsing JSON output...
JSON output parsed in: 7ms
The file _output/gitlogg.json was saved!
Wallaces-iMac:gitlogg Wallace$
```

Maybe there's some left-over code lying around? [This line](https://github.com/Inventitech/gitlogg/blob/8b2219ddd5bb28b1b342ed4f2ffbf9d3db3ce527/scripts/output-intermediate-gitlog.sh#L15) also seem displaced.

**Comment by dreamyguy** [comment]
I'll merge the patch for the `xargs` problem, but I'm changing the base at my end to **develop** instead of **master** branch. I do this everywhere else and might as well start doing it here. 🎸

**Comment by Inventitech** [comment]
Oops, I think something went wrong while rebasing :-).

**Comment by Inventitech** [comment]
Have a look now, please.

**Comment by dreamyguy** [comment]
I've decided to merge to a dedicated branch instead, to keep develop relese-able. Will be testing it asap.

**Pull request merged**